packages/orbit-components/src/Itinerary/consts.ts

```diff
@@ -0,0 +1,85 @@
+export enum STATUS_TYPES {
```

Comment:
I've created this file to store enums that were placed in `stories.tsx` file. The reason behind this is that `PALETTER_COLORS` contains so many lines.